docs/asset_precompilation.md

```diff
@@ -0,0 +1,30 @@
+## Asset Checksumming
+
+`bin/asset_checksum`
+
+In order to make it easy to tell if the assets need to be recompiled or if the cached versions can be used, we generate a checksum based on the content of `app/assets` (including client theme files). The goal is for the checksum to be guaranteed to change if a change is made to the source assets that will alter the compiled output, *and* guaranteed *not* to change otherwise.
```

Comment:
Double checking "client theme files" are the files we store on S3 and pull down in prior to checksumming right?

Reply:
Yes, the ones in the `openpath-ecs-assets` S3 bucket. We should have a better nomenclature to differentiate those... thoughts? 

I can also put the name of the bucket here, but it would be good to have an easier way of referring to the two different things: (a) client theme files/assets (`openpath-ecs-assets`) and (b) client compiled asset (cache?) (`openpath-precompiled-assets`).

Reply:
I took a stab at a glossary at the top. Feel free to adjust.